README.md

```diff
@@ -1,6 +1,6 @@
 # ckanext-canada
 
-[![Tests](https://github.com/open-data/ckanext-canada/workflows/Tests/badge.svg?branch=master)](https://github.com/open-data/ckanext-canada/actions?query=name%3ATests)
+[![Tests](https://github.com/open-data/ckanext-canada/workflows/Tests/badge.svg?branch=canada-v2.9&event=push)](https://github.com/open-data/ckanext-canada/actions?query=name%3ATests)
```

Comment:
looks like something we'll forget to change :thinking:

Reply:
Changed to master branch. People will get emails for tests failing in canada-v2.9 anyways, so should not need visual badge for this specific branch.